src/main/webapp/view/transcript/transcript.js

```diff
@@ -163,6 +163,26 @@ function isWithinCurrentTimeRange(currentTimeMs) {
       currentTimeMs <= currentTranscriptLine.endTimestampMs;
 }
 
+/**
+ * Returns true if `currentTimeMs` is before the starting timestamp of
+ * the first transcript line.
+ */
+function isBeforeTheFirstTranscriptLine(currentTimeMs) {
+  const firstTranscriptLine = document.getElementsByTagName('li')[0];
+  return currentTimeMs < firstTranscriptLine.startTimestampMs;
+}
+
+/**
+ * Returns the closest transcript line based on `currentTimeMs`.
+ */
+function getNextTranscriptLine(currentTimeMs) {
+  // TODO: Implement this function to get
+  // the next line based on search.
+  currentTranscriptLine = currentTranscriptLine.nextElementSibling;
+  console.log(currentTimeMs);
```

Comment:
Is this needed?

Reply:
Sort of, I needed to use the parameter for the checks to pass but I'll remove it when I implement the function.

Reply:
Makes sense thanks